### PR TITLE
fix: wrong sync behavior when removing all prev panels

### LINKED ISSUE #598: In Next js, Cannot read properties of undefined (reading 'nativeElement')
## Description
Next js Link를 통해서 같은 페이지의 다른 id로 이동하는 경우에 발생하는 버그입니다.

Bug that occurs when moving to another id on the same page through Nextjs Link.

image
### Simple code
```tsx
import Flicking from '@egjs/react-flicking'
import Card from 'component/Card'

const CardSwiper = ({ card, id }: any) => {
  const { data: cards } = useQuery(`/card/${id}`) // react query

  return cards && cards.length > 0 ? (
    <Flicking>
      {cards.map((card) => (
        <div>
          <Card card={card} />
        </div>
      ))}
    </Flicking>
  ) : (
    <></>
  )
}
```
### 아래 조건이 모두 해당될 때 발생합니다.
1. sample/:id 의 페이지 첫 진입시 id에 따라 api fetching으로 가져온 데이터를 기준으로 swiper 내부 노드를 렌더링함 (ex: get api/sample/:id)
2. 1번 페이지에서 Next Link를 통해 id가 다른 sample/:id 로 이동 (ex: sample/1 -> sampel/2)

## Tried
1. 다른 페이지에서 sample/:id 진입하는 경우, url을 직접 입력해서 이동하는 경우에는 발생하지 않습니다.
2. Next Link를 통해 이동하기 전에 해당 데이터를 undefined나 null로 초기화 후에 이동하는 경우는 발생하지 않습니다. 하지만 뒤로가기를 누르면 다시 발생합니다.

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#598

## Details
This fixes #598

### CODE REVIEW

**Comment by mixed** [comment]
LGTM. There seems to be an issue with the coverage measurement, but if it is resolved, there is no problem.

File: src/cfc/sync.ts [hunk 1]
```diff
@@ -56,13 +59,15 @@ export default (flicking: Flicking, diffResult: DiffResult<any>, rendered: any[]
     let startIdx = -1;
     let prevIdx = -1;
 
+    const addedElements = rendered.slice(prevList.length);
```
Comment (daybrush) [inline src/cfc/sync.ts:62]: Is this code correct?

```
1 2 3 4 
=>
1 5 2 6 3 7 4
```

Comment (WoodNeck) [reply src/cfc/sync.ts:62]: As we're adding the newly added panels at the last, yes.
- https://github.com/naver/egjs-flicking/blob/master/src/cfc/getRenderingPanels.ts#L22

**Review by daybrush — APPROVED** [review]
LGTM

**Comment by WoodNeck** [comment]
Fixed broken cfc coverage send

**Review by ahn3407 — APPROVED** [review]
LGTM

**Pull request merged**